__tests__/app.test.js

```diff
@@ -15,3 +23,31 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("2. GET /api/topics", () => {
+  test("2a. 200: responds with array of topcs including slug & description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(Array.isArray(topics)).toBe(true);
```

Comment:
We don't really need this expect statement, as we're doing a length check and forEach below, so it implicitly shows it's an array 👍